src/realm/lang_bind_helper.hpp

```diff
@@ -100,6 +100,8 @@ class LangBindHelper {
     static const LinkViewRef& get_linklist_ptr(Row&, size_t col_ndx);
     static void unbind_linklist_ptr(const LinkViewRef&);
 
+    static Table& get_backlink_parent(const Table*, size_t col_ndx);
```

Comment:
`target` sounds like the opposite direction (we have `get_link_target()` to follow links). I change it to `table`.